packages/tokamak/sdk/src/interfaces/types.ts

```diff
@@ -37,7 +37,7 @@ export enum L2ChainID {
   ZORA_MAINNET = 7777777,
   MODE_SEPOLIA = 919,
   MODE_MAINNET = 34443,
-  THANOS_SEPOLIA_TEST = 111551118080,
+  THANOS_SEPOLIA_TEST = 111551119090,
```

Comment:
Can you delete the TEST? (THANOS_SEPOLIA_TEST -> THANOS_SEPOLIA)

Reply:
Done